### PR TITLE
Update doctrine inflector

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -47,12 +47,14 @@ jobs:
                       elasticsearch-package-constraint: '^5.0'
 
                     - php-version: '7.4'
+                      add-text: 'Allow to fail: '
```
Comment (niklasnatter) [inline .github/workflows/test-application.yaml]: would call this `job-name-prefix` 😇

Comment (alexander-schranz) [reply .github/workflows/test-application.yaml]: Sounds good for me ;)

File: Generator/DocumentGenerator.php [hunk 2]
```diff
@@ -81,6 +82,11 @@ public function __construct()
 <fields>
 }';
 
+    /**
+     * @var Inflector
+     */
+    private $inflector;
```
Comment (niklasnatter) [inline Generator/DocumentGenerator.php:88]: is this set anywhere? 🤔

Comment (niklasnatter) [reply Generator/DocumentGenerator.php:88]: okay you are setting it in the `getInflector` to only trigger the deprecation once. is okay for me 👍

**Pull request merged**